bee-network/src/event.rs

```diff
@@ -0,0 +1,37 @@
+// Copyright 2021 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+//! A module that deals with events published by the network (layer).
+
+use crate::peer::ConnectedPeer;
+
+/// Represents a network event.
+pub enum NetworkEvent {
+    /// Fired when a peer has been successfully connected and handshaked.
+    PeerConnected(ConnectedPeer),
+    /// This is event
```

Comment:
punctuation
also maybe better describe the event?

Reply:
This is what happens when you add random docs to suppress the warnings hahaha